resources/migrations/000_legacy_migrations.yaml

```diff
@@ -5829,16 +5829,14 @@ databaseChangeLog:
   - changeSet:
         id: 109
         author: camsaul
-        comment: Added 0.34.2
```

Comment:
Since we're changing the checksum we should probably add `validCheckSum: ANY`, even if real world instances are ignoring this file now.

Reply:
This was a tip from @nvoxland actually:

> Alternately, you can set dbms="none" to keep it in the file but not have it actually apply anywhere. That doesn't change the checksum

(I left that exchange in the commit message)

Reply:
TIL! I thought Liquibase was far more aggressive with the checkum